test/e2e/e2e_suite_test.go

```diff
@@ -107,6 +105,70 @@ func TestRegistryCreation(t *testing.T) {
 			}
 			return ctx
 		}).
+		Assess("Verify the OwnerReference deletion", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			// Get all the VulnerabilityReport/Image/SBOM associated with the Registry
+			labelSelector := labels.FormatLabels(
+				map[string]string{handlers.LabelManagedByKey: handlers.LabelManagedByValue},
+			)
+			images := storagev1alpha1.ImageList{}
+			err := wait.For(conditions.New(cfg.Client().Resources()).ResourceListN(
```

Comment:
We are using Testify for the other tests. We should use it here as well instead of relying on the idiom:
```
if err != nil {
  t.Error(err)
}
```

Reply:
e.g.`require.NoEror(t, err)`